pam/integration-tests/helpers_test.go

```diff
@@ -32,7 +32,20 @@ var (
 	authdArtifactsDirSync sync.Once
 )
 
-func runAuthd(t *testing.T, gpasswdOutput, groupsFile string, currentUserAsRoot bool) string {
+type authdInstance struct {
+	mu        sync.Mutex
+	instances uint64
+	socket    string
+	gPasswd   string
+	groups    string
```

Comment:
These are a bit vague... Do you mind updating them to indicate better what they mean or add a comment for them?

Reply:
Changed the naming, let me know if it's better.